samples/net/ws_console/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _websocket-server-sample:
```

Comment:
whoops, just notice in https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/3894/3/tests that this document label is already in use (must be unique across all the documents)